tests/Support/DatabaseMigrations.php

```diff
@@ -0,0 +1,16 @@
+<?php
+trait DatabaseMigrations
+{
+    /**
+     * @before
+     */
+    public function runDatabaseMigrations()
+    {
+        $this->artisan('migrate');
+        $this->artisan('db:seed');
+
+        $this->beforeApplicationDestroyed(function () {
+            $this->artisan('migrate:rollback');
```

Comment:
Not sure if we want to destroy. Currently the seeds are created with development in mind, not tests. I did the seeding because each time I modify the DB schema, I need to re-populate the DB.